### PR TITLE
add winui sentry app

### PR DESCRIPTION
**Pull request by stevenbrix**
this PR adds a simple WinUI sentry example app using the new `swift-*` repos for the bindings:

- [swift-winui](https://github.com/thebrowsercompany/swift-winui)
- [swift-windowsappsdk](https://github.com/thebrowsercompany/swift-windowsappsdk)

these repos not only contain the bindings, but a few helpers for making starting the app even easier as it takes care of intializing the windows app runtime and providing default IXamlMetadataProvider implementation

### CODE REVIEW

**Review by compnerd — APPROVED** [review]

**Review by brianmichel — APPROVED** [review]
Hell yeah!

File: Examples/SentryExampleWinUI/Application.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Foundation
```
Comment (brianmichel) [inline Examples/SentryExampleWinUI/Application.swift:1]: Needs copyright notice

File: Examples/SentryExampleWinUI/SentryConfiguration.swift [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+enum SentryConfiguration {
+  // Update this with a DSN from Sentry to allow for configuration
+  static let dsn = "https://key@sentry/project"
+}
```
Comment (brianmichel) [inline Examples/SentryExampleWinUI/SentryConfiguration.swift:6]: Missing newline

File: Package.swift [hunk 3]
```diff
@@ -79,6 +86,9 @@ package.dependencies += [
     // This revision is important since it's the first one before the swift-win32 repo moved to versioned symlinks
     // for different swift-tools-versions.
     .package(url: "https://github.com/compnerd/swift-win32", revision: "07e91e67e86f173743329c6753d9e66ac4727830"),
+    .package(url: "https://github.com/thebrowsercompany/swift-windowsfoundation", branch: "main"),
+    .package(url: "https://github.com/thebrowsercompany/swift-winui", branch: "main"),
+    .package(url: "https://github.com/thebrowsercompany/swift-windowsappsdk", branch: "main"),
```
[the remark this answers is not in the document]
Comment (stevenbrix) [reply Package.swift:91]: yeah i'm holding off on that until they are

[the remark this answers is not in the document]
Comment (stevenbrix) [reply Package.swift]: they are public!

**Pull request merged**